samples/WebApiContrib.Core.Samples/Controllers/VersionedController.cs

```diff
@@ -0,0 +1,15 @@
+using Microsoft.AspNetCore.Mvc;
+using WebApiContrib.Core.Versioning;
+using WebApiContrib.Core.Samples.Model;
+
+namespace WebApiContrib.Core.Samples.Controllers
+{
+    [Route("api/versioned")]
+    public class VersionedController : Controller
+    {
+        public IActionResult Get()
+        {
+            return new VersionedObjectResult(new PersonModel("Kristian", "Hellang", age: 27));
```

Comment:
Nah, I just thought it would be nice to have a way to opt-in to the versioning. There's an option, `RequireVersionedObjectResult`, that's set to `true` by default. This makes sure the versioning negotiation (and finding a mapper for the model) is skipped if the result isn't a `VersionedObjectResult`. Do you think we should just skip it and work with `ObjectResult`?